plugins/vpc-branch-eni/plugin/commands.go

```diff
@@ -280,9 +280,18 @@ func (plugin *Plugin) createVLANLink(branch *eni.Branch, linkName string, ipAddr
 			return err
 		}
 
+		// If gatewayIPAddress is nil, that means the gateway ip is not provided in NetConfig. In that case,
```

Comment:
Please use "gateway IP address", not just "ip". Two more occurrences in log statements below.

Reply:
updated